Appraisals

```diff
@@ -1,40 +1,26 @@
-case RUBY_VERSION
-
-when '2.3.3', '2.5.3', '2.6.3' then
-
-  appraise "ruby-#{RUBY_VERSION}-rails507" do
-    gem 'rails', '5.0.7'
-    gem 'grape', '1.1.0'
-    gem 'rails-controller-testing'
-  end
-
-  appraise "ruby-#{RUBY_VERSION}-rails516" do
-    gem 'rails', '5.1.6'
-    gem 'grape', '1.2.3'
-    gem 'rails-controller-testing'
-  end
+RAILS_VERSIONS = ['5.2.2.1', '6.0.2.1']
+GRAPE_VERSIONS = ['1.1.0', '1.2.3', '1.3.0']
 
-  appraise "ruby-#{RUBY_VERSION}-rails521" do
-    gem 'rails', '5.2.1'
-    gem 'grape', '1.2.3'
-    gem 'rails-controller-testing'
-  end
-
-  appraise "ruby-#{RUBY_VERSION}-rails522" do
-    gem 'rails', '5.2.2'
-    gem 'grape', '1.2.3'
-    gem 'rails-controller-testing'
-  end
+case RUBY_VERSION
 
-  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.5.0')
-    appraise "ruby-#{RUBY_VERSION}-rails6" do
-      gem 'rails', '~> 6.0'
-      gem 'grape', '1.2.3'
-      gem 'rails-controller-testing'
-      gem 'sqlite3', '~> 1.4'
+when '2.5.3', '2.6.3' then
```

Comment:
I am only seeing this gem used in `ops_app` and `property`, which both use ruby 2.6.3, so support for ruby 2.5.3 can probably be dropped as well at this point. (Unless it is used somewhere else that I am missing)

Reply:
Additionally, the `.ruby-version` file still lists `ruby-2.3.3` so that can probably be updated to `ruby-2.6.3`

Reply:
Just realized that this is a public gem, so I guess it should support more versions than one's used at Appfolio.

Reply:
Yeah, we should support more than just what we use internally.

Reply:
Yeah, as a public repo we need to support more versions. I can update the `.ruby-version` though.